back/engines/free/email_campaigns/app/models/email_campaigns/builders/internal_comment_campaign_commands_builder.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module EmailCampaigns
+  class Builders::InternalCommentCampaignCommandsBuilder
```

Comment:
Good idea with this builder. Just one little remark.

As it's not a model, I would rather put this file into the `services` folder.